vlib/v/fmt/tests/maps_expected.vv

```diff
@@ -25,3 +25,9 @@ headers := http.new_header_from_map({
 	.content_type:  'application/json'
 	.authorization: 'Bearer abcdef'
 })
+
+utf8_keys := {
+	'ß':        's'
+	'В начале': 'В начале'
+	'最初有道': '最初有道'
```

Comment:
This might not look perfectly aligned depending on the charset. Tho the spacing here is the same when checked in the terminal. If character widths are rendered differently, it's probably nothing that can be addressed.

Reply:
The problem is that many of the Asian characters are up to 1.5 "regular" chars in width.  But they still only count as a sing when counting the number.